Formula/onlykey-agent.rb

```diff
@@ -0,0 +1,212 @@
+class OnlykeyAgent < Formula
+  include Language::Python::Virtualenv
+
+  desc "Middleware that lets you use OnlyKey as a hardware SSH/GPG device"
+  homepage "https://docs.crp.to/onlykey-agent.html"
+  url "https://files.pythonhosted.org/packages/02/c1/27c6cfbc5ee63fca91e37915d0182c0bfb988ca12362f01bcd5451e0ee10/onlykey-agent-1.1.14.tar.gz"
+  sha256 "096f20580ae112f57c1b9b279ed17728dc8e6f0fca301be586b9d976177f1523"
+  license "LGPL-3.0-only"
+
+  # libusb and gnupg according to https://docs.crp.to/onlykey-agent.html#installation
+  depends_on "gnupg@2.2"
```

Comment:
Why not the modern version?

Reply:
onlykey-agent 1.1.14 works with gnupg2.3 and gnupg2.2 (LTS version)

Reply:
Then it should use the newer version

Reply:
This was addressed in 271acd112276cc046f1328fb0ba367b7d446896f by using bare `gnupg` per a brew audit recommendation.